### PR TITLE
Bug fix yum install

### PR DESCRIPTION
**Pull request by etsauer**

### CODE REVIEW

File: provisioning/osc-install [hunk 1]
```diff
@@ -213,6 +213,10 @@ do_prep() {
   yum -y update &>/dev/null || error_out "Failed to run yum update during Prep phase."
```
Comment (oybed) [inline provisioning/osc-install:214]: Do you also want to include "yum clean all" above this line (for the master)?

File: provisioning/lib/helpers [hunk 2]
```diff
@@ -9,10 +9,11 @@ get_hostnames() {
   # Can reverse lookup hostname for an ip with
   # dig -x 172.16.166.11 | sed -ne 's/.*\(node[0-9]*.ose.example.com\).*/\1/p'
   if [ -n "$3" ]; then
-    dns_server="@${3}"
+    local nameserver="@${3}"
   fi
   for ip in ${1//,/ }; do
-    hostnames="${hostnames},$(dig -x $dns_server $ip | sed -n "s/.*\(\(master\|node\)[0-9]*.${2}\).*/\1/p" | sort | head -n1)"
+    #echo "dig $dns_server -x $ip"
```
Comment (oybed) [inline provisioning/lib/helpers]: Same as other PR - do you want to remove the commented out line?

**Comment by oybed** [comment]
@etsauer just a couple of questions, other than looks good

**Comment by etsauer** [comment]
@oybed made the suggested changes.

**Pull request merged**